doc/source/ray-core/placement-group.rst

```diff
@@ -3,38 +3,43 @@ Placement Groups
 
 .. _ray-placement-group-doc-ref:
 
-Placement groups allow users to atomically reserve groups of resources across multiple nodes (i.e., gang scheduling). They can be then used to schedule Ray tasks and actors packed as close as possible for locality (PACK), or spread apart (SPREAD). Placement groups are generally used for gang-scheduling actors, but also support tasks.
+Placement groups allow users to atomically reserve groups of resources across multiple nodes. The groups then can be used to schedule Ray tasks and actors packed as close as possible for locality (`PACK` and `STRICT_PACK`), or spread apart (`SPREAD` and `STRICT_SPREAD`).
 
-Java demo code in this documentation can be found here `<https://github.com/ray-project/ray/blob/master/java/test/src/main/java/io/ray/docdemo/PlacementGroupDemo.java>`__.
+See the :ref:`Java Placement Groups demo`<https://github.com/ray-project/ray/blob/master/java/test/src/main/java/io/ray/docdemo/PlacementGroupDemo.java> for more information.
 
-Here are some use cases:
+The Placement Groups guide covers these topics: 
+* :ref:`Use Cases`<use-cases>
+* :ref:`Key Concepts`<key-concepts>
+...
+* :ref:`Lifecycle`<lifecycle>
+* :ref:`Fault Tolerance`<fault-tolerance>
 
-- **Gang Scheduling**: Your application requires all tasks/actors to be scheduled and start at the same time.
-- **Maximizing data locality**: You'd like to place or schedule your actors close to your data to avoid object transfer overheads.
+Use cases
+---------
+
+- **Gang Scheduling**: All tasks or actors can be scheduled to start at the same time.
+- **Maximize data locality**: Place or schedule your actors close to your data to avoid object transfer overheads.
 
```

Comment:
Are there considerations that can be added here? Benefits and Considerations for each case? Or, is the chosen function entirely dependent on the customer already knowing what they need to do?